### ISSUE TITLE
Spatial reference error box is persistent even after corrected.

### ISSUE THREAD

**Issue by ACueva** [Esri/distance-direction-addin-dotnet#524]
Environment tested on:
ArcMap 10.6

Steps to reproduce:

Start with a blank map, DO NOT ADD A BASEMAP.

From the line tab, enter the following:
Starting Point: 34.73N 116.948W
Ending Point: 60.236869N 24.895304E

Observe the following error message about the spatial reference:
![2018-02-26_14-18-03](https://user-images.githubusercontent.com/3093940/36698868-df2be60a-1aff-11e8-96cd-451e28a96eb1.png)

Add a basemap.
![2018-02-26_14-21-56](https://user-images.githubusercontent.com/3093940/36699039-6b13eb9a-1b00-11e8-9e09-5aaf7db89242.png)

Error persists until either enter is pressed or ending point values are cleared out and re-entered.

**Comment by kgonzago** [comment]
After some digging, without a basemap present, the supplied coordinates for the starting and ending points are still valid (the CC parser categorizes it as DD). The validation error message was produced by the SDK and not one of our custom messages. The exception message is thrown during the creation of a geodetic line. A workaround can be applied to check if a layer is present with a valid spatial reference.

**Comment by topowright** [comment]
@kgonzago I would not call that a work around. The tool is designed to draw a graphic on a map and it needs a map in order to draw that line.

**Comment by kgonzago** [comment]
@topowright Current logic allows users to create a line graphic without the presence of a basemap. I added additional logic to handle for that scenario.

A listener was implemented to check for layers added to the map. This will re-validate the the `End Point`. If a valid spatial reference is present, the validation error will disappear. 

Fix is in this [PR](https://github.com/Esri/distance-direction-addin-dotnet/pull/553)
